fix(apps): exit steam big picture mode on session end

eaglesemanation opened the pull request:
## Description
This will make example Steam Big Picture app automatically exit big picture mode on disconnect.

Note: only validated on Linux. Don't think that's really an issue, considering it's just a url change


### Screenshot
<!--- Include screenshots if the changes are UI-related. --->


### Issues Fixed or Closed
Adds command from ["Command to quit steam big picture"](https://github.com/orgs/LizardByte/discussions/206) discussion


## Type of Change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [x] Documentation update (changes to documentation)
- [ ] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

ReenigneArcher commented:
Thank you for the PR!

We should probably do this by default in our apps.json files. Can you update these as well?

- https://github.com/LizardByte/Sunshine/blob/master/src_assets/linux/assets/apps.json
- https://github.com/LizardByte/Sunshine/blob/master/src_assets/macos/assets/apps.json
- https://github.com/LizardByte/Sunshine/blob/master/src_assets/windows/assets/apps.json

eaglesemanation commented:
Updated, also replaced `open steam <url>` with `open <url>` in MacOS docs, because that's how it is in `apps.json`. Not sure which one is right though, and I don't have a device to test it on

eaglesemanation commented:
This doesn't actually close steam, just converts back to desktop mode from Big Picture mode
